data/RS3LAB/projects.yaml

```diff
@@ -150,5 +150,17 @@ projects:
       - Concurrency
       - Fuzzing
       - Server
+    code:
+      type: Lab GitHub
+      url: https://github.com/sslab-gatech/hydra
+      date_last_commit: 2022-08-08
+    information:
+      - type: Paper
+        title: Finding Semantic Bugs in File Systems with an Extensible Fuzzing Framework
+        url: https://sanidhya.github.io/pubs/2019/hydra.pdf
```

Comment:
Usually we link to the infoscience.epfl.ch, but it doesn't seem to be there, so this is OK.